.github/codecov.yml

```diff
@@ -0,0 +1,20 @@
+comment:
+  layout: "header, diff, components"
+
+component_management:
+  individual_components:
+    - component_id: cvat-ui
+      name: cvat-ui
+      paths:
+        - cvat-canvas/**
```

Comment:
what is about [cvat-canvas3d](https://github.com/opencv/cvat/tree/develop/cvat-canvas3d)?